packages/pages/src/App.js

```diff
@@ -59,57 +62,68 @@ export default provideState({
     id: "",
   }),
   effects: {
-    async initialize() {
-      await this.effects.refreshEntries();
+    initialize() {
+      const t = this;
```

Comment:
I had to add more variables like 'ts' for 'this.state' or 't' for 'this', because in the 'websocket', 'this' is unknown.. Is there an other option ?